content/ai/concepts/computer-vision/computer_vision.md

```diff
@@ -0,0 +1,86 @@
+---
+Title: 'AI for Computer Vision'
+Description: 'Computer vision is a subfield of artificial intelligence, which specialises in recognition, extraction and processing of image data to simulate human vision. '
```

Comment:
Make this smaller and descriptive. You can elaborate the definition after the meta data.